Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM frolvlad/alpine-glibc
```

Comment:
We discussed tagging this in case it changes under our feet

I have no idea how likely that is...it feels unlikely...but maybe no reason not to tag it?!